### PR TITLE
[WIP] UI Store

### PR DESCRIPTION
**Pull request by yasatbilgin**

### CODE REVIEW

File: src/ui/instance.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+        currentCategory: null,
+        currentAccount: null,
+        currentContact: null,
+        filterIsOn: false,
+        lastLocation: null,
+        loggedIn: false,
+        allowedFileIds: null
+    })
+
+    inACall = computed(() => this.state.currentContact != null)
```
Comment (digi604) [inline src/ui/instance.js:38]: convert to getter

File: src/utils/file.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+function isExpiredOrNotReady(now, startDate, endDate) {
```
Comment (digi604) [inline src/utils/file.js:1]: not sure this function should be inside utils/file

File: src/utils/file.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    if (typeof startDate === 'undefined' || typeof endDate === 'undefined') return false
+    if (startDate == null || endDate == null) return false
+    if (startDate == 0 || endDate == 0) return false
+    return startDate > now || endDate < now
+}
+
+function showInUI(now, file) {
```
Comment (digi604) [inline src/utils/file.js:8]: rename function to shouldDisplay

Comment (digi604) [reply src/utils/file.js:8]: both functions are not exportet?

**Comment by digi604** [comment]
Also maybe it would be a good idea to split the store. 1 for navigation / current category. Using the params store for current account, user and one store for files? Not sure... but maybe we could optimize this? And is this all compatible with connect?

**Comment by digi604** [comment]
as small test for just initializing and testing the store would not decrease coverage this much.

**Comment by digi604** [comment]
you can mock the fireEvent calls no?

**Comment by digi604** [comment]
or just initialize the store with some dummy data and see if all the getters work.

**Comment by elibolonur** [comment]
Closing due to usage of new branch

**Pull request closed without merging**